adr/ADR-32.md

```diff
@@ -172,6 +175,7 @@ Service API does not have any assumptions about API spec or schema formats.
     name: string,
     id: string,
     version: string,
+    metadata: Record<string,string>;      
```

Comment:
What is Record<string,string>, sorry not everyone is familiar with this construct.

Reply:
It's just an associative array in typescript. Aka map, hash, dict etc

Reply:
Same as stream and consumer metadata

Reply:
Definitely need to use one language to describe all ADRs :)